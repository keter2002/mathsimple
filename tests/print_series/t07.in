1 3 ln\(x\)
