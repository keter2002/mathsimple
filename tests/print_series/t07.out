x|log|
0.00000 0.69315 1.09861
